machine/plexxi/px_switch2/kernel/config

```diff
@@ -0,0 +1,37 @@
+CONFIG_VIRTIO_BLK=y
+CONFIG_VIRTIO_NET=y
+CONFIG_HVC_DRIVER=y
+CONFIG_VIRTIO_CONSOLE=y
+CONFIG_HW_RANDOM_VIRTIO=y
+CONFIG_VIRTIO=y
+CONFIG_VIRTIO_RING=y
+CONFIG_VIRTIO_PCI=y
+CONFIG_VIRTIO_BALLOON=y
+CONFIG_PARAVIRT_GUEST=y
+CONFIG_PARAVIRT=y
+CONFIG_PARAVIRT_SPINLOCKS=y
+CONFIG_PARAVIRT_CLOCK=y
+CONFIG_KVM_CLOCK=y
+CONFIG_KVM_GUEST=y
+CONFIG_HAVE_KVM=y
```

Comment:
As for the old code submitted previously -- it was never pulled into the ONIE repo, so it does not exist in the project.  Please base your changes off the ONIE master branch when making a pull request.

See item 2 here:
https://github.com/opencomputeproject/onie/wiki/Contributing#general-patching-philosophy